script/DeployMocks.s.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: BUSL 1.1
+// Valorem Labs Inc. (c) 2023.
+pragma solidity 0.8.16;
+
+import "forge-std/Script.sol";
+
+// import "../src/TokenURIGenerator.sol";
+import "test/mocks/MockUSDC.sol";
+import "test/mocks/MockWETH.sol";
+import "test/mocks/MockGMX.sol";
+
+
+// Salt for mocks
+
+contract DeployMocksScript is Script {
+        uint256 constant SALT = 0x2023091301;
```

Comment:
Let's add a basic `forge` command to run this script, as a comment, eg, https://github.com/valorem-labs-inc/clear/blob/master/script/Deploy.s.sol#L11